src/chain/network_event.rs

```diff
@@ -80,8 +80,12 @@ impl EventSigPayload {
 
 #[derive(Debug, Serialize, Deserialize, PartialEq, Eq, PartialOrd, Ord, Hash, Clone)]
 pub struct OnlinePayload {
+    // Identifier of the joining node.
     pub p2p_node: P2pNode,
+    // The age the node should have after joining.
     pub age: u8,
+    // The knowledge of the joining node about the destination section.
```

Comment:
what kind of knowledge, according to a later on usage, it seems the version of the section?
why not just name the field as: dst_sec_version if so?

Reply:
I agree that `knowledge` is a bit vague, but it's already established term - we use it in `their_knowledge` in `SharedState` and this field is basically the same thing (the node's knowledge of out section). We can maybe try to come up with better name but we should be consistent.

Reply:
Maybe could use `their_knowledge` here too?

Reply:
Yeah, sounds reasonable. Will do.